frontend/src/index.jsx

```diff
@@ -8,16 +8,20 @@ import ThemeContextProvider from "./context/ThemeContext";
 import { MainContainer } from "./MainContainer";
 import { BowtieVersionContextProvider } from "./context/BowtieVersionContext";
 import { DragAndDrop } from "./components/DragAndDrop/DragAndDrop";
-import { parseReportData } from "./data/parseReportData";
+import {
+  parseReportData,
+  parseImplementationData,
+} from "./data/parseReportData";
 import URI from "urijs";
+import { ImplementationReportView } from "./components/ImplementationReportView/ImplementationReportView";
 
 const reportHost =
   import.meta.env.MODE === "development"
     ? "https://bowtie.report"
     : window.location.href;
 const reportUri = new URI(reportHost).directory(import.meta.env.BASE_URL);
 
-const dialectToName = {
+export const dialectToName = {
```

Comment:
Given this and the mapping (which I mentioned above I think belongs in some more central place) I'd probably recommend creating a `Dialect` object at some point containing the various things we want to capture about them (human readable names, mappings to and from their URIs, etc.). Feel free to leave that for now though.

Reply:
Done!